server/routes/foods.js

```diff
@@ -9,6 +9,15 @@ router.get('/', (req, res) => {
     .then(foods => res.status(200).json(foods))
 })
 
+router.post('/', (req, res) => {
```

Comment:
Same here all edit routes below the get routes please :)

Reply:
done